v0.1/CSharpSwagger/CSharpSwagger/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using IO.Swagger.Api;
+using IO.Swagger.Client;
+using IO.Swagger.Model;
+using Newtonsoft.Json;
+
+namespace CSharpSwagger
+{
+	class MainClass
+	{
+		public static void Main (string[] args)
+		{
+			Console.WriteLine("Hello World");
+			ApiClient client = new ApiClient("http://192.168.250.28:5003");
+			DeedApi deed_get = new DeedApi(client);
+			DefaultApi deed_post = new DefaultApi (client);
+			Console.WriteLine ("Client created");
+			Console.WriteLine(" ");
+			Console.WriteLine ("Creating Post to Client:");
+
+			IO.Swagger.Model.DeedApplication deed_app = new DeedApplication ();
+
+			deed_app.TitleNumber = "DT567568";
+			deed_app.MdRef = "e-MD123G";
+
+			Console.WriteLine ("Creating Borrower");
+
+//			{
```

Comment:
Leaving this comment here means that we are going to have to support it, i.e. if the schema changes we need to remember to come in and change this comment.

Surely it is the purpose of the swagger docs to do what you said?